website/docs/internals/credentials-helpers.mdx

```diff
@@ -1,37 +1,37 @@
 ---
 description: >-
   Credentials helpers are external programs that know how to store and retrieve
-  API tokens for remote OpenTF services.
+  API tokens for remote OpenTofu services.
 ---
 
 # Credentials Helpers
 
-For OpenTF-specific features that interact with remote network services,
-such as [module registries](/docs/registry), OpenTF by default looks for
+For OpenTofu-specific features that interact with remote network services,
+such as [module registries](/docs/registry), OpenTofu by default looks for
 API credentials to use in these calls in
 [the CLI configuration](/docs/cli/config/config-file).
 
 Credentials helpers offer an alternative approach that allows you to customize
-how OpenTF obtains credentials using an external program, which can then
+how OpenTofu obtains credentials using an external program, which can then
 directly access an existing secrets management system in your organization.
 
 This page is about how to write and install a credentials helper. To learn
 how to configure a credentials helper that was already installed, see
 [the CLI config Credentials Helpers section](/docs/cli/config/config-file#credentials-helpers).
 
-## How OpenTF finds Credentials Helpers
+## How OpenTofu finds Credentials Helpers
 
 A credentials helper is a normal executable program that is installed in a
 particular location and whose name follows a specific naming convention.
 
 A credentials helper called "credstore", for example, would be implemented as
 an executable program named `terraform-credentials-credstore` (with an `.exe`
 extension on Windows only), and installed in one of the
-[default plugin search locations](/docs/plugin/how-opentf-works#plugin-locations).
+[default plugin search locations](/docs/plugin/how-tofu-works#plugin-locations).
```

Comment:
We don't have these docs yet, but I think the final path would use full name: `opentofu` here